sycl/doc/EnvironmentVariables.md

```diff
@@ -14,7 +14,7 @@ subject to change. Do not rely on these variables in production code.
 | SYCL_PI_TRACE | Described [below](#sycl_pi_trace-options)  | Enable specified level of tracing for PI. |
 | SYCL_BE | PI_OPENCL, PI_LEVEL_ZERO, PI_CUDA | Force SYCL RT to consider only devices of the specified backend during the device selection. |
 | SYCL_DEVICE_TYPE | One of: CPU, GPU, ACC, HOST | Force SYCL to use the specified device type. If unset, default selection rules are applied. If set to any unlisted value, this control has no effect. If the requested device type is not found, a `cl::sycl::runtime_error` exception is thrown. If a non-default device selector is used, a device must satisfy both the selector and this control to be chosen. This control only has effect on devices created with a selector. |
-| SYCL_DEVICE_FILTER (name TBD) | {backend:device_type:device_num} | Force SYCL RT to use the specified device with optional backend and device_num information. It can list any number of triples separated by commas. This env var affects all different device selectors and device discovery. When no device that can satisfy the triple by a default_selector, a heuristic will choose the device that has the closest match. Possible values of device_type are \*,host,cpu,gpu,acc. Possible values of backend are \*,opencl, level_zero, cuda. Device_num is an unsigned integer that indexes the enumeration of devices from sycl::platform::get_device() call. All triple entries are optional, but one of them should be present. For example, to use cpu and level_zero gpu device number 0, one can set SYCL_DEVICE_FILTER=cpu,level_zero:gpu::0. This environment variable will limit loading only specified plugins into the SYCL RT. For example, SYCL_DEVICE_FILTER=level_zero and cpu_selector will throw an error because the CPU device is not supported by the level_zero backend. |
+| SYCL_DEVICE_FILTER (name TBD) | {backend:device_type:device_num} | Force SYCL RT to use the specified device with optional backend and device_num information. It can list any number of triples separated by commas. This env var affects all different device selectors and device discovery. When no device that can satisfy the triple by a default_selector, a heuristic will choose the device that has the closest match. HOST device is always available for the default_selecotr and the filter won't affect HOST device availability in any case. Possible values of device_type are \*,host,cpu,gpu,acc. Possible values of backend are \*,opencl, level_zero, cuda. Device_num is an unsigned integer that indexes the enumeration of devices from sycl::platform::get_device() call. All triple entries are optional, but one of them should be present. For example, to use cpu and level_zero gpu device number 0, one can set SYCL_DEVICE_FILTER=cpu,level_zero:gpu::0. This environment variable will limit loading only specified plugins into the SYCL RT. For example, SYCL_DEVICE_FILTER=level_zero and cpu_selector will throw an error because the CPU device is not supported by the level_zero backend. |
```

Comment:
Can you remove vague "When no device that can satisfy the triple by a default_selector, a heuristic will choose the device that has the closest match." in favor of more specific (I still don't like the behavior) "HOST device is always available for the default_selecotr and the filter won't affect HOST device availability in any case."?

Reply:
Spelling: default_selecotr > default_selector

Reply:
The default_selector is based on heuristic point. And lots of people asked how the filter affects this behavior, so I added this sentence to clarify. The confusion on the user side is that the user requested default_selector(cpu_device), and he gets a gpu device based on the heuristic point. That's the current default behavior, and the filter won't change this behavior.
The sentence about HOST device is not enough to explain this behavior.

Reply:
> Spelling: default_selecotr > default_selector

Thanks. I fixed it.